src/main/resources/templates/email/ltft-updated/v1.0.0.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <title>LTFT Application Status Update</title>
+</head>
+<body>
+<h1>Email Message</h1>
+<h2>Subject</h2>
+<th:block th:fragment="subject">Your LTFT Application Status Update</th:block>
+<h2>Content</h2>
+<th:block th:fragment="content">
+  <div style="text-align:right">
+    <img style="margin-right:25px" src='https://trainee.tis.nhs.uk/nhse-logo.png' alt="NHS England logo"/>
+  </div>
+  <p>Dear Dr Gilliam,</p>
```

Comment:
Without wanting to jump the gun on a PR that is still being worked on, I think this might be the source of the failing integration tests. I think you'd need something more like 
`<p th:replace="~{fragments/greeting/v1.0.0 :: p}"></p>`
(assuming you have the standard name fields in the object you are using to populate the template).